src/backend/static.jl

```diff
@@ -1,35 +1,54 @@
 function _to_static(::Type{ComputationalGraphs.Sum}, subgraphs::Vector{Graph{F,W}}, subgraph_factors::Vector{F}) where {F,W}
     if length(subgraphs) == 1
-        return "(g$(subgraphs[1].id) * $(subgraph_factors[1]))"
+        factor_str = subgraph_factors[1] == 1 ? "" : " * $(subgraph_factors[1])"
+        return "(g$(subgraphs[1].id)$factor_str)"
     else
-        return "(" * join(["g$(g.id) * $gfactor" for (g, gfactor) in zip(subgraphs, subgraph_factors)], " + ") * ")"
+        terms = ["g$(g.id)" * (gfactor == 1 ? "" : " * $gfactor") for (g, gfactor) in zip(subgraphs, subgraph_factors)]
+        return "(" * join(terms, " + ") * ")"
     end
 end
 
 function _to_static(::Type{ComputationalGraphs.Prod}, subgraphs::Vector{Graph{F,W}}, subgraph_factors::Vector{F}) where {F,W}
     if length(subgraphs) == 1
-        return "(g$(subgraphs[1].id))"
+        factor_str = subgraph_factors[1] == 1 ? "" : " * $(subgraph_factors[1])"
+        return "(g$(subgraphs[1].id)$factor_str)"
     else
-        return "(" * join(["g$(g.id)" for g in subgraphs], " * ") * ")"
+        terms = ["g$(g.id)" * (gfactor == 1 ? "" : " * $gfactor") for (g, gfactor) in zip(subgraphs, subgraph_factors)]
+        return "(" * join(terms, " * ") * ")"
+        # return "(" * join(["g$(g.id)" for g in subgraphs], " * ") * ")"
     end
 end
 
+function _to_static(::Type{ComputationalGraphs.Power{N}}, subgraphs::Vector{Graph{F,W}}, subgraph_factors::Vector{F}) where {N,F,W}
+    factor_str = subgraph_factors[1] == 1 ? "" : " * $(subgraph_factors[1])"
+    return "((g$(subgraphs[1].id))^$N$factor_str)"
```

Comment:
I suggest moving the " * " from `factor_str` to the return statement for readability everywhere in `static.jl`. This way it is clearer at a glance what each expression compiles to. For example,
```julia
factor_str = subgraph_factors[1] == 1 ? "" : "$(subgraph_factors[1])"
return "((g$(subgraphs[1].id))^$N * $factor_str)"
```

Reply:
Thank you for your suggestion. But "*" should be absent in this expression when subgraphs_factors[1]==1. I changed it this way mainly to reduce one redundant multiplication operation.

Reply:
Ah, of course, I missed this somehow.